cli/tests/pcluster/api/controllers/test_cluster_compute_fleet_controller.py

```diff
@@ -278,6 +377,14 @@ def _send_test_request(self, client, cluster_name="clustername", region="us-east
             ("slurm", "START_REQUESTED", datetime.now()),
             ("slurm", "PROTECTED", datetime.now()),
             ("slurm", "UNKNOWN", None),
+            ("plugin", "RUNNING", datetime.now()),
+            ("plugin", "STOPPED", datetime.now()),
+            ("plugin", "STOPPING", datetime.now()),
+            ("plugin", "STARTING", datetime.now()),
+            ("plugin", "STOP_REQUESTED", datetime.now()),
+            ("plugin", "START_REQUESTED", datetime.now()),
+            ("plugin", "PROTECTED", datetime.now()),
+            ("plugin", "UNKNOWN", None),
```

Comment:
does it add value to test all combinations also for the plugin scheduler? especially since unit tests are not extremely fast.